### DISCUSSION TITLE
schema.inputObjectType reference issue

### DISCUSSION THREAD

**Discussion by TheoMer** [prisma/prisma#3080] [Q&A] [answered]
So I have two inputObjectType's:

```
schema.inputObjectType({
  name: 'SizeCreateInput',
  definition(t) {
    t.string('id'),
    t.string('name'),
    t.string('label')
  }
})

schema.inputObjectType({
  name: 'SizeWhereUniqueInput',
  definition(t) {
    t.string('id'),
    t.string('name')
  }
})
```

which I want to reference from a third object as such:

```
schema.inputObjectType({
  name: 'SizeCreateOneInput',
  definition(t) {
    create: SizeCreateInput
    connect: SizeWhereUniqueInput
  }
})
```

How do I correctly specify the create and connect parameters to reference SizeCreateInput and SizeWhereUniqueInput respectively?

**Comment by Weakky** [comment, marked as answer]
Hey @TheoMer,

use `t.field` to reference another input object type.

```ts
definition(t) {
  t.field('create', { type: 'SizeCreateInput' })
  t.field('connect', { type: 'SizeWhereUniqueInput' })
}
```

**Reply by TheoMer** [reply]
@Weakky Many thanks!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
